apps/wallet-dashboard/components/dialogs/unstake/views/UnstakeView.tsx

```diff
@@ -74,7 +77,7 @@ export function UnstakeView({
     } = delegatedStakeDataResult;
 
     const delegationId = extendedStake?.stakedIotaId;
-    const isPreparingUnstake = !unstakeData || isUnstakeTxPending;
+    const isNotEnoughGas = error && error.message.includes(NOT_ENOUGH_BALANCE_ID);
```

Comment:
this has only been tested with having no balance at all, but it can also happen that there is balance but is too low 
![image](https://github.com/user-attachments/assets/60dfe024-57ea-40bf-895c-c73032135eaf)
same in the wallet

Reply:
you can reproduce this error by having a tiny bit of dust